IFC4x3/PropertyEnumerations/i/PEnum_InjectionPackers.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_InjectionPackers" Name="PEnum_InjectionPackers" UniqueId="e0b1e967-ae6b-4fde-8b86-1a26bb30621e">
```

Comment:
Rename to "PEnum_InjectionPackerType"?

Reply:
Name changed to PEnum_InjectionPackerType.